websites/M/MEE6/presence.ts

```diff
@@ -2,59 +2,208 @@ const presence = new Presence({
+										presenceData.details = "Loading";
+										presenceData.smallImageKey = Assets.Loading;
+									} else {
+										presenceData.details = "Editing command";
+										presenceData.state = command.textContent.slice(4);
+									}
+								} else {
+									presenceData.details = "Viewing module";
+									presenceData.state = await capitalizeFirstLetter(
+										document.querySelector(
+											'[class="font-bold text-dark-100 text-h5"]'
+										)?.textContent ?? pathnameSplit[4].trim()
+									);
+								}
+							}
+						}
+						break;
+					}
+					case "tutorials": {
+						switch (pathnameSplit[3]) {
+							// eslint-disable-next-line no-undefined
+							case undefined: {
+								presenceData.details = "Viewing all tutorials";
+								break;
+							}
+							default: {
+								presenceData.details = "Reading tutorials about:";
+								presenceData.state =
+									document.querySelector(
+										'div[dir="ltr"]'
+									)?.firstElementChild?.textContent;
+								presenceData.smallImageKey = Assets.Reading;
+							}
+						}
+						break;
+					}
+					default: {
+						presenceData.details = strings.browse;
+					}
+				}
+			}
+			break;
+		}
+		case "help.mee6.xyz": {
+			const search = document.querySelector<HTMLInputElement>(
+				'[class="special ui-autocomplete-input"]'
 			);
-			presenceData.details = "Dashboard - Editing plugin:";
-			presenceData.state = title.textContent;
-			presenceData.smallImageKey = "writing";
-		} else if (document.location.pathname.includes("/dashboard/")) {
-			title = document.querySelector(".subHeaderMenuListItem.selected");
-			presenceData.details = "Dashboard - Viewing tab:";
-			presenceData.state = title.textContent;
-		} else if (document.location.pathname.includes("/premium")) {
-			presenceData.details = "Reading about premium";
-			presenceData.smallImageKey = "reading";
-		} else if (document.location.pathname === "/")
-			presenceData.details = "Viewing the homepage";
-	} else if (document.location.hostname === "help.mee6.xyz") {
-		title = document.querySelector<HTMLElement>("head > title");
-		search = document.querySelector<HTMLInputElement>(
-			"body > header > div.csh-wrapper > form > span > input"
-		);
-		if (!search.textContent) {
-			presenceData.details = "Helpdesk searching for:";
-			presenceData.state = search.value;
-			presenceData.smallImageKey = "searching";
-		} else if (title.textContent === "MEE6 Helpdesk")
-			presenceData.details = "Browsing the helpdesk";
-		else {
-			presenceData.details = "Helpdesk viewing:";
-			presenceData.state = title.textContent.replace(" | MEE6 Helpdesk", "");
-			presenceData.smallImageKey = "reading";
+			if (search?.value) {
+				presenceData.details = strings.search;
+				presenceData.state = search.value;
+				presenceData.smallImageKey = Assets.Search;
+			} else if (document.querySelector('[class="article__title"]')) {
+				presenceData.details = "Reading support article about:";
```

Comment:
Remove `:` from strings
```suggestion
				presenceData.details = "Reading support article about";
```